sql/setup.sql

```diff
@@ -19,7 +19,7 @@ CREATE TABLE tips (
 );
 
 INSERT INTO users (email, password_hash) VALUES
-('admin@behuman.com', '$2y$04$F8I5POR7CFJR1HgpKeuvs.jYH6Ge60paGorKoQxvlhzWjoZoZGkBi');
+('behumannotbot@gmail.com', '$2y$04$F8I5POR7CFJR1HgpKeuvs.jYH6Ge60paGorKoQxvlhzWjoZoZGkBi');
```

Comment:
Should this be protected and placed in .dotenv file?

Reply:
Good question - I had that thought too. Shouldn't be an issue as you can not log in with the token. You log in on the front end with the password that the token represents. However, still worth asking about!